compiler/tests/testSuite.bal

```diff
@@ -57,6 +57,66 @@ function testCompileEU(string path, string kind) returns file:Error|io:Error? {
     }
 }
 
+function moduleDir(string filePath) returns string|file:Error? {
+    string subModPath = filePath.substring(0, filePath.length()-4) + ".modules";
+    if check file:test(subModPath, file:IS_DIR) {
+        return check file:normalizePath(subModPath, file:CLEAN);
+    }
+}
+
+function verifyErrorLineNumber(CompileError err, string path) returns file:Error? {
```

Comment:
This is all so complicated. Write a function

```
function expectedErrorLocation(string testCasePath) returns [string, int]?
```

Then have do asserts that the filename in the error matches the expected filename (probably trim some directories off first), and if there's a line number in the error, that it matches the expected line number.

Reply:
fixed